src/main/java/com/flagsmith/FlagsmithClient.java

```diff
@@ -398,7 +416,36 @@ public Builder setDefaultFlagPredicate(@NonNull Predicate<String> defaultFlagPre
      */
     public Builder setDefaultFlagValueFunction(
         @NonNull Function<String, String> defaultFlagValueFunction) {
-      this.client.defaultFlagValueFunc = defaultFlagValueFunction;
+      this.configuration.flagsmithFlagDefaults.setDefaultFlagValueFunc(defaultFlagValueFunction);
+      return this;
+    }
+
+    /**
+     * When a flag does not exist in Flagsmith or there is an error, the SDK will return an empty
+     * list of flags by default. For example:
+     * <ul>
+     *   <li>if you call identifyUserWithTraits(...) and the call fails</li>
+     *   <li>this method will return an empty list of flags</li>
+     * </ul>
+     *
+     * <p>If you would like the SDK to return a default list of flags with
+     * default values, you can set the default flag names with this method. For example:
+     * <ul>
```

Comment:
As per the other comment thread, I don't think this makes sense as a list?

Reply:
Switch to paragraph style explanation